src/main/java/Task.java

```diff
@@ -0,0 +1,27 @@
+public class Task {
+    public String description;
+    public boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toString(){
+        String output = "[ ] ";
+        if(isDone){
+            output = "[X] ";
+        }
+
+        return output + description;
```

Comment:
use this.description to get description to clarify since its not a local variable